lib/Bio/EnsEMBL/DataCheck/Checks/DescriptionDisplayXrefs.pm

```diff
@@ -0,0 +1,77 @@
+=head1 LICENSE
+
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::DescriptionDisplayXrefs;
```

Comment:
I think a better name might be 'XrefFormat' - provides a succinct summary of what we're testing.

Reply:
Having suggested that, it sounded like it rang a bell, and there is already a datacheck of that name, covering the same ground. So the tests here should be added to that, rather than creating a new one.